tensorflow/inference/docker/2.3.0/py37/Dockerfile.eia

```diff
@@ -7,8 +7,8 @@ LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 LABEL dlc_major_version="1"
```

Comment:
The `dlc_major_version` should be incremented to `2` as we are upgrading the python version.